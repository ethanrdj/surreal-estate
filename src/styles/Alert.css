.alert-error {
  color: red;
  background: #ffe5e5;
  padding: 8px 20px 8px 20px;
  border-style: solid;
  border-color: red;
  border-radius: 5px;
  border-width: 0.2px;
  margin-bottom: 10px;
}

.alert-success {
  color: green;
  background: #f4fef4;
  padding: 8px 20px 8px 20px;
  border-style: solid;
  border-color: green;
  border-radius: 5px;
  border-width: 0.1px;
  margin-bottom: 10px;
}
