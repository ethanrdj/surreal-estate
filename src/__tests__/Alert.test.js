import React from "react";
import { render } from "@testing-library/react";
import Alert from "../components/Alert";

describe("Alert", () => {
  it("displays error message", () => {
    const { getByText } = render(<Alert message="Error!" />);

    expect(getByText(/Error/).textContent).toBe("Error!");
  });

  it("displays success message", () => {
    const { getByText } = render(<Alert message="Success!!!" success />);

    expect(getByText(/Success/).textContent).toBe("Success!!!");
  });

  it("does not render is message is not truthy", () => {
    const { asFragment } = render(<Alert message="" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
