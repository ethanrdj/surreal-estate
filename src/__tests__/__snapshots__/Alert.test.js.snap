// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert does not render is message is not truthy 1`] = `<DocumentFragment />`;
